### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by CathyF9600**
enlarged search bar with carousel background

### CODE REVIEW

**Review by RobertRen1122 — APPROVED** [review]
Looks nice :)

**Comment by CatherineZM** [comment]
The components look good! Please clean up the code. There are a lot of commented-out parts.

File: frontend/src/Components/HomePage/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      <div className="col-md-6 col-lg-6 col-11 mx-auto my-auto search-box">
+        <div className="input-group form container">
+          <input type="text"
+                 className="form-control search-input"
+                 name="search"
+                 onChange={this.handleChange}
+                 placeholder="Search..."
+                 color="white"
+                 // autofocus="autofocus"
+                 // autocomplete="off"
```
Comment (CatherineZM) [inline frontend/src/Components/HomePage/SearchBar.js]: Clean up the comments

File: frontend/src/HomePage.js [hunk 2]
```diff
@@ -5,13 +5,16 @@ import MainPsg from './Components/HomePage/MainPsg';
 import MainPsgSide from './Components/HomePage/MainPsgSide';
 import MsgBox from "./Components/Misc/MsgBox";
 import {mainDomain} from "./configuration";
-import Footer from "./Components/HomePage/Footer";
+// import SearchBar from "./Components/HomePage/SearchBar";
+import NavBar from "./Components/Misc/NavBar";
+import BootstrapCarousel from "./Components/HomePage/BootstrapCarousel";
+// import Footer from "./Components/HomePage/Footer";
```
Comment (CatherineZM) [inline frontend/src/HomePage.js]: footer was commented out?

File: frontend/src/styles/BootstrapCarousel.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.container-fluid {
+  /*object-fit: cover;*/
+  /*margin-top: 20vh;*/
```
Comment (CatherineZM) [inline frontend/src/styles/BootstrapCarousel.css]: Clean up comments as well

File: frontend/src/styles/SearchBar.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+/*section{*/
+/*    align-items: center;*/
+/*}*/
+
+/*.box{*/
+/*    position: absolute;*/
+/*    top:70%;*/
+/*    left: 50%;*/
+/*    border-radius: 5px;*/
+/*    background-color: rgba(250, 247, 247, 0.75);*/
```
Comment (CatherineZM) [inline frontend/src/styles/SearchBar.css]: Same here

**Review by CatherineZM — CHANGES_REQUESTED** [review]

**Review by CatherineZM — APPROVED** [review]

**Pull request merged**